test-utils/runtime/src/lib.rs

```diff
@@ -717,6 +721,17 @@ impl_runtime_apis! {
 			None
 		}
 	}
+
+	#[cfg(not(feature = "disable-genesis-builder"))]
+	impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
+		fn create_default_config() -> Vec<u8> {
+			create_default_config::<RuntimeGenesisConfig>()
```

Comment:
```suggestion
			create_default_config::<RuntimeGenesisConfig>()
```

Please don't use the word `config` as a shorthand for `GenesisConfig` anywhere, as it is super confusing, given that there exists something entirely different in the runtime with the same name (`trait Config`). 

As an example, the word "default config" is being typically used to describe #13454. 

There is also https://github.com/paritytech/substrate/issues/14462.

Reply:
I see your point. On the other hand the context is clearly given by `GenesisBuilder` trait. Refer also to this [comment](https://github.com/paritytech/substrate/pull/14131#pullrequestreview-1435453355).

Maybe I should simplify methods' names and go this way?
```rust
trait GenesisConfigBuilder {
  fn get_default(..);
  fn build(..);
};
```

Reply:
@kianenigma I will do clean-up in follow-up: https://github.com/paritytech/substrate/issues/14621